src/runner.ts

```diff
@@ -173,25 +185,80 @@ async function runLinter(options: Options, logger: Logger): Promise<LintResult>
     return doLinting(options, files, program, logger);
 }
 
-function resolveFilesAndProgram(
-    { files, project, exclude, outputAbsolutePaths }: Options,
+/** Read the stdin stream into a string with all of the contents. */
+async function readStdin(): Promise<string> {
```

Comment:
`runner.ts` is getting pretty huge. Please separate this standalone utility to a different file (`src/utils.ts, maybe?).